.github/workflows/performance-benchmarks.yml

```diff
@@ -1,12 +1,6 @@
 name: Performance Benchmarks
 on:
-  push:
-    branches:
-      - master
-      - release-**
-  pull_request:
-    branches:
-      - master
+  workflow_call:
```

Comment:
I don't know if there's any way for us to do this differently, but with CircleCI, we actually check out the web app  and set up Cypress before waiting for the build to complete. Since we're now running this all after the build completes, this'll take slightly longer now than it did before

Reply:
We could indeed run the Cypress setup, prior to artifact generation , that would save about 70 seconds. 
<img width="1663" alt="Screenshot 2023-02-09 at 11 11 27" src="https://user-images.githubusercontent.com/36594156/217768047-78455f3f-ed3f-4605-b2d8-3b11e66dad82.png">


But I'd suggest keeping the workflow as simple as possible, instead of constantly probing GitHub API to detect job completed. 

The gains are minimal (1 minute), and after all, the execution time of the `Jest Tests` is taking longer than that. Therefore no actual gains are presented.

Reply:
As discussed, I strongly recommend keeping the pipeline simple, instead of constantly probing GitHub API to detect job completed. 
This is recommended as best practise, keeping the pipeline simple, instead of introducing complexity for questionable gains. 

Let me know if we are aligned.

Reply:
Yep, this works for me